app/models/miq_group.rb

```diff
@@ -230,8 +230,18 @@ def validate_default_tenant
     end
   end
 
+  def single_group_users?
+    group_user_ids = user_ids
+    users.includes(:miq_groups).where(:id => group_user_ids).where.not(:miq_groups => {:id => id}).count != group_user_ids.size
```

Comment:
Looks good to me, but we might want tests just for this method to ensure it does the right thing in a few different scenarios.